pnpm-lock.yaml

```diff
@@ -1364,6 +1364,8 @@ importers:
         specifier: ^0.33.0
         version: 0.33.0(@vitest/ui@0.33.0)(happy-dom@10.5.1)
 
+  sdk/move-binary-format/pkg: {}
```

Comment:
I keep seeing this flip flop. Curious why this happens.

Reply:
I think this happens because there aren't checked in, and if you run an install without generating the packages, this won't be part of the lockfile